peer/lifecycle/lifecycle.proto

```diff
@@ -161,6 +161,26 @@ message CheckCommitReadinessResult{
     map<string, bool> approvals = 1;
 }
 
+// QueryApprovalForMyOrgArgs is the message used as arguments to
+// `_lifecycle.QueryApprovalForMyOrg`.
+message QueryApprovalForMyOrgArgs {
+    string name = 1;
+    int64 sequence = 2;
+}
+
+// QueryApprovalForMyOrgResult is the message returned by
+// `_lifecycle.QueryApprovalForMyOrg`.
+message QueryApprovalForMyOrgResult {
+    int64 sequence = 1;
+    string version = 2;
+    string endorsement_plugin = 3;
+    string validation_plugin = 4;
+    bytes validation_parameter = 5;
+    protos.CollectionConfigPackage collections = 6;
+    bool init_required = 7;
+    string package_id = 8;
+}
+
```

Comment:
Do we really need two new messages here?
These are similar to `QueryChaincodeDefinitionArgs` and `QueryChaincodeDefinitionResult` respectively.
Maybe it's ok to have two request messages, but it seems like `QueryApprovalForMyOrgResult ` should be reduced to a single message `QueryChaincodeDefinitionResult`

Reply:
Unfortunately, I think we do need both of these messages.

You'll notice that `QueryChaincodeDefinitionResult` has `approvals`, which, doesn't really make sense in this context, and is missing `package_id` which will vary from org to org.

Reply:
Yeah ... I noticed the differences ... but it seems like a waste to entirely duplicate things ... but I guess it does not hurt and keeps things loosely coupled